plugins/modules/ipam_subnet.py

```diff
@@ -2436,6 +2436,18 @@ def __init__(self, *args, **kwargs):
         exclude = ["state", "csp_url", "api_key", "id"]
         self._payload_params = {k: v for k, v in self.params.items() if v is not None and k not in exclude}
         self._payload = Subnet.from_dict(self._payload_params)
+        # Safely remove unwanted attributes
+        if self._payload.dhcp_config:
+            self._payload.dhcp_config.abandoned_reclaim_time = None
```

Comment:
The Documentation and Return for subnet should to be updated to remove these 3 attributes.

Reply:
all the fields are required to be sent as part of the inheritance payload

Reply:
Yes , but that is a bug which is scheduled to be resolved later !

Reply:
As discussed on call, its a backend bug and needs all field to be added as part of inheritance